ext/storage_sources/local_store/local_store.c

```diff
@@ -715,9 +717,9 @@ local_flush_finish(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session,
         goto err;
 
     local->op_count++;
-    if ((ret = rename(source, dest_path)) != 0) {
+    if ((ret = link(source, dest_path)) != 0) {
```

Comment:
For the rename->link change, I would add a comment to the effect that the "link" could be replaced by a file copy if that is ever needed for portability.

Reply:
Fixed.